test/cli/fixtures/expected/tap-outputs.js

```diff
@@ -80,5 +80,11 @@ ok 5 A-Test > derp
 	"qunit --reporter does-not-exist": `No reporter found matching "does-not-exist".
 Available reporters from JS Reporters are: console, tap
 Available custom reporters from dependencies are: npm-reporter
+`,
+
+	/* eslint-disable max-len */
+	"qunit hanging-test": `Error: Process exited before tests finished running
+Last test to run has an async hold. Ensure all assert.async() callbacks are invoked and Promises resolve. You should also set a standard timeout via QUnit.config.testTimeout.
 `
+	/* eslint-enable */
```

Comment:
This should probably say `/* eslint-enable max-len */`, otherwise any other rules disabled with `/* eslint-disable */` before this line would be re-enabled with this line.